libs/dataset-explorer/src/lib/getDatasetOption.ts

```diff
@@ -20,5 +21,12 @@ export function getDatasetOption(
   if (chartProps?.chartType === ChartTypes.Scatter) {
     return getDatasetScatterOption(jointData, plotlyProps, chartProps);
   }
+  if (
+    chartProps?.yAxis.property &&
```

Comment:
nit: maybe move 
```
chartProps?.yAxis.property
```
to a separate variable to simplify:
```
const yAxisProp = chartProps?.yAxis.property
if (yAxisProperty && jointData.metaDict[yAxisProp].isCategorical || jointData.metaDict[yAxisProp].treatAsCategorical)
...
```

Reply:
Updated, thanks Ilya!